### PR TITLE
GH-16: update header ui

### LINKED ISSUE #16: update header of web app so that it matches design
add logo, robot name and guide button

### PR DESCRIPTION
**Pull request by alex-waverley**
Now UI of header matches design
current ui https://prnt.sc/pjGDnhyd8vYh
Closes #16

### CODE REVIEW

File: frontend/src/components/header/index.js [hunk 1]
```diff
@@ -30,6 +35,10 @@ const Header = observer(() => {
     setActiveTab(newValue);
   };
 
+  const handeGuideClick = () => {
```
Comment (sskorol) [inline frontend/src/components/header/index.js:38]: Did you mean `handle`?

File: frontend/src/components/roboDesignation/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import {ReactComponent as Robologo} from './spot.svg'
+import './style.css'
+
+const RoboDesignation = () => {
+  return (
+    <div className='robo-designation'><Robologo className='robo-designation__logo'/>Spot 1234 869 Full Name</div>
```
Comment (sskorol) [inline frontend/src/components/roboDesignation/index.js:7]: Let's move a text into a store with a default value, as it'll be changed on a settings screen.

**Review by sskorol — CHANGES_REQUESTED** [review]

File: frontend/src/components/header/index.js (hunk 1 above)
Comment (alex-waverley) [reply frontend/src/components/header/index.js]: yep, thanks, done

File: frontend/src/components/roboDesignation/index.js (hunk 2 above)
Comment (alex-waverley) [reply frontend/src/components/roboDesignation/index.js:7]: the store will be created within another issue #17

**Review by skorol-waverley — APPROVED** [review]

**Review by sskorol — APPROVED** [review]

**Pull request merged**